.github/workflows/master.yml

```diff
@@ -0,0 +1,63 @@
+# Push to master workflow.
+#
+# Runs when a PR has been merged to the master branch.
+#
+# 1. Generates a release build.
+# 2. If the last commit is a version change, publish.
+
+name: Master
+
+on:
+ push:
+  branches:
+   - master
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+      - uses: actions/checkout@v1
+      # TODO: Measure if any speedup from caching (no Cargo.lock, so always reuse cache)
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --verbose --release
+      # Get information for the publish.
+      - shell: bash
+        id: commit_message
+        run: |
+          commit_message=$(git log --format=%B -n 1 ${{ github.sha }})
+          echo "::set-output name=commit_message::$commit_message"
```

Comment:
I think you need to include this in the `publish` job. You can't reference information from steps in other jobs. I ran into this yesterday with SCL.

It also makes me wonder if the condition at the job level will work, since you won't have the commit message before the job starts. I will be able to verify this with SCL.